src/Dynamo/src/HyparDynamo/HyparDynamo.csproj

```diff
@@ -31,14 +30,15 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\Revit\RevitHyparTools\RevitHyparTools.csproj" />
+    <ProjectReference Include="..\..\..\Elements\Elements.csproj" />
```

Comment:
This should not be a local reference right? This should point to `Elements`?

Reply:
all this work is inside the Elements repo, and this reference points to the Elements.csproj inside the Elements repo that this project is in.... 
a bit confusing but basically I'm point to an "adjacent" elements.csproj in the same repo, which I think is right, this should always be using the latest elements no?

Reply:
I see now. The way you’ve done it is correct.